src/utils/tsLocaleKeysFunctionBuilder.js

```diff
@@ -161,6 +163,19 @@ class TSLocaleKeysFunctionBuilder {
         templateFile = templateFile.replace(/\bILocaleKeysTemplate\b/g, this.getInterfaceName(this.functionName.trim()));
         templateFile = templateFile.replace('/* constructor args */', this.getConstructorArgs());
         templateFile = templateFile.replace('/* placeholder: keys here */', this.getKeys(localeKeys));
+
+        if (this.reactHook) {
```

Comment:
Maybe having a shorthand inline if, instead of repeating the `/* placeholder: react import here */`? 
i think it feels cleaner. 
wdyt?

Reply:
not sure I got your point